spec/factories/prisons.rb

```diff
@@ -26,6 +26,10 @@
       '01154960%03d' % n
     end
 
+    adult_age do
+      10 + rand(9)
```

Comment:
I want it to randomly generate children or adults to ensure that all assumptions are clearly stated in tests.